### ISSUE TITLE
IE11: No black outline for keyboard focused items in ComboBox/Dropdown

### ISSUE THREAD

**Issue by ayonix** [microsoft/fluentui#21820]
### Environment Information

- **Package version(s)**: 8.55.3
- **Browser and OS versions**: Internet Explorer 11 (or Outlook), Windows 10

### Please provide a reproduction of the bug in a codepen:

https://fabricweb.z5.web.core.windows.net/oufr/8.55.3/#/examples/ComboBox
https://fabricweb.z5.web.core.windows.net/oufr/8.55.3/#/examples/dropdown

#### Actual behavior:
By changing the item via keyboard input (arrow up/down) the item has no outline to reflect that.
![image](https://user-images.githubusercontent.com/1388039/154837704-5bc32873-b8f8-4bba-a146-85fcab04bccd.png)

#### Expected behavior:
The element should have an outline, same as in Chrome/Firefox.
![image](https://user-images.githubusercontent.com/1388039/154837738-8d61c4a3-74fb-4a30-b5ce-e4df2bc1a161.png)

### Priorities and help requested:

Are you willing to submit a PR to fix? No, because I can't set up the development environment at the moment. The fix of #16938 seems to apply here too.

Requested priority: Normal

Products/sites affected: any fluentui/react addin using dropdown/combobox that is used in IE11 or Outlook for example.

**Comment by spmonahan** [comment]
As we've [sunset Internet Explorer support](https://github.com/microsoft/fluentui/wiki/Internet-Explorer-11-Sunset) I'm closing this issue.
